crazyswarm2/src/teleop.cpp

```diff
@@ -39,18 +39,28 @@ class TeleopNode : public rclcpp::Node
     TeleopNode()
         : Node("teleop")
     {
+        
         subscription_ = this->create_subscription<sensor_msgs::msg::Joy>(
-            "joy", 1, std::bind(&TeleopNode::joyChanged, this, _1));
+            "joy", 1, std::bind(&TeleopNode::joyChangedAxis, this, _1));
```

Comment:
with that the old functionality (button presses) won't work anymore. I suggest keeping joyChanged as function here and copy the logic that you currently have in joyChangedAxis to joyChanged.